src/dispatch.rs

```diff
@@ -169,10 +171,12 @@ impl<'t> Dispatcher<'t> {
 
         #[cfg(target_os = "emscripten")]
         self.dispatch_seq(res);
+
+        self.dispatch_thread_local(res);
```

Comment:
I think that this method is actually needed public, because it separates control of thread local systems and others. In my case i don't want to execute thread local systems all the time after executing parallel, but some times I want to execute only them on demand. In that case I could create separate dispatcher, but with this little method it's easier.